Crashlytics/Crashlytics/Models/FIRCLSSettings.h

```diff
@@ -14,7 +14,11 @@
 
 #import <Foundation/Foundation.h>
 
-#import "FBLPromise.h"
+#if __has_include(<FBLPromises/FBLPromises.h>)
+#import <FBLPromises/FBLPromises.h>
+#else
+#import "FBLPromises.h"
```

Comment:
This changed from `FBLPromise` to `FBLPromises` - is that intentional? Same as below.

Reply:
Yeah we're being consistent and using the umbrella header in all cases